lib/v2/reuters/common.js

```diff
@@ -5,23 +5,37 @@ const { art } = require('@/utils/render');
 const path = require('path');
 
 module.exports = async (ctx) => {
+    const MUST_FETCH_BY_TOPICS = ['authors'];
+    const CAN_USE_SOPHI = ['world'];
+
     const category = ctx.params.category;
     const topic = ctx.params.topic ?? (category === 'authors' ? 'reuters' : '');
     const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 20;
+    const useSophi = ctx.query.sophi === 'yes' && 'topic' !== '' && CAN_USE_SOPHI.includes(category);
```

Comment:
```suggestion
    const useSophi = ctx.query.sophi === 'true' && 'topic' !== '' && CAN_USE_SOPHI.includes(category);
```

Reply:
https://github.com/DIYgod/RSSHub/blob/bb820b029ade79e70d1aa2a267453dc017efb04c/lib/v2/thepaper/utils.js#L21 uses `yes`, is that OK?